clustering/jgroups/extension/src/main/java/org/jboss/as/clustering/jgroups/subsystem/TransportResourceDefinition.java

```diff
@@ -211,77 +148,10 @@ private TransportResourceDefinition(Parameters parameters, UnaryOperator<Resourc
     public ManagementResourceRegistration register(ManagementResourceRegistration parent) {
         ManagementResourceRegistration registration = super.register(parent);
 
-        new WriteAttributeStepHandler(new WriteThreadingAttributeStepHandlerDescriptor()) {
-            @Override
-            protected void validateUpdatedModel(OperationContext context, Resource model) throws OperationFailedException {
-                // Add a new step to validate instead of doing it directly in this method.
-                // This allows a composite op to change both attributes and then the
-                // validation occurs after both have done their work.
-                context.addStep(new OperationStepHandler() {
-                    @Override
-                    public void execute(OperationContext context, ModelNode operation) throws OperationFailedException {
-                        ModelNode conf = context.readResource(PathAddress.EMPTY_ADDRESS).getModel();
-                        // TODO doesn't cover the admin-only modes
-                        if (context.getProcessType().isServer()) {
-                            for (ThreadingAttribute attribute : EnumSet.allOf(ThreadingAttribute.class)) {
-                                if (conf.hasDefined(attribute.getName())) {
-                                    // That is not supported.
-                                    throw new OperationFailedException(JGroupsLogger.ROOT_LOGGER.threadsAttributesUsedInRuntime());
```

Comment:
Please comment out `org.jboss.as.clustering.jgroups.logging.JGroupsLogger#threadsAttributesUsedInRuntime` as it's no longer used.

Reply:
Fixed.